src/main/kotlin/io/github/rybalkinsd/kohttp/ext/RequestExt.kt

```diff
@@ -0,0 +1,38 @@
+package io.github.rybalkinsd.kohttp.ext
+
+import okhttp3.Request
+import okio.Buffer
+import java.nio.charset.Charset
+
+/**
+ *  Build curl command of the request
+ *
+ * @author doyaaaaaken
+ */
+internal fun Request.buildCurlCommand(): String {
+    return StringBuilder().apply {
+        append("curl -X ${method()}")
+
+        //headers
+        val headers = headers()
+        for (i in 0 until headers.size()) {
+            val name = headers.name(i)
+            var value = headers.value(i)
+            if (value[0] == '"' && value[value.length - 1] == '"') {
```

Comment:
this segment of code looks rather complicated for me, probably it will be easier with the sequence, or will need further simplification

please check `.last()` and `.first()` methods